bin/cqlsh.py

```diff
@@ -665,7 +665,7 @@ def maybe_warn_py2(self):
         py2_suppress_warn = 'CQLSH_NO_WARN_PY2'
         if sys.version_info[0:2] == (2, 7) and not os.environ.get(py2_suppress_warn):
             print("Python 2.7 support is deprecated. "
-                  "Install Python 3.6+ or set %s to suppress this message.\n" % (py2_suppress_warn,))
+                  "Install Python 3.8+ or set %s to suppress this message.\n" % (py2_suppress_warn,))
```

Comment:
I think Python 2.7 doesn't work at all now. Cqlsh will immediately fail with a syntax error before ever reaching this check. So I think we can remove the entire `maybe_warn_py2` check and its associated test, `test_warn_py2`.

Reply:
That's a good point. Honestly I wanted to keep this test and see whether it will break in CI but then I realise given https://jenkins-stargazer.aws.dsinternal.org/job/ds-cassandra-pr-gate/job/PR-1960/9/testReport/cqlshlib.python3.8.jdk11.no-cython.x86_64.test.test_cqlsh_output/TestCqlshOutput/ isn't run against python 2.7 (as the config has been removed) it won't be run at all.
And yes you are right, it will fail much earlier in the call-stack so it doesn't make much sense to keep it.